config/default.yml

```diff
@@ -671,3 +671,25 @@ Rails/HttpStatus:
   - symbolic
   VersionAdded: '1.23'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/HttpStatus
+
+Rails/PersistenceCalledOutsideExample:
+  Description: Checks for persistence calls outside example blocks.
+  Enabled: true
```

Comment:
I think we need to ass `Safe: false` as well, as it would produce false positives